src/components/note-list.tsx

```diff
@@ -135,26 +142,11 @@ export function NoteList({ baseQuery = "" }: NoteListProps) {
                   disableTooltip
                   aria-label="Change view"
                   className="h-11 w-11 rounded-md bg-bg-secondary hover:bg-bg-tertiary coarse:h-12 coarse:w-12"
+                  onClick={() => toggleView()}
                 >
                   <ViewTypeIcon viewType={viewType} />
                 </IconButton>
               </DropdownMenu.Trigger>
```

Comment:
You can remove the all the `DropdownMenu.*` components here, since we only need the `IconButton` now.